### ISSUE TITLE
`roman-numerals` tests has expected and actual swapped

### ISSUE THREAD

**Issue by paulmr** [exercism/xerlang#154]
If you have a bad result while running eunit on the roman-numerals exercise, the error is reported with the expected value and the actual value swapped around, which can be confusing when you are dealing with roman numerals where it isn't always easy to spot the correct one.

For example, if I change the test result for 48 to `"OOPS"` I get this:

```
3> eunit:test(roman_numerals_tests).
roman_numerals_tests: convert_48_test...*failed*
in function roman_numerals_tests:'-expect_roman/2-fun-0-'/2 (/meta/p/pmr/src/ext/xerlang/exercises/roman-numerals/_build/test/lib/roman_numerals/test/roman_numerals_tests.erl, line 9)
**error:{assertEqual,[{module,roman_numerals_tests},
              {line,9},
              {expression,"Expected"},
              {expected,"OOPS"},
              {value,"XLVIII"}]}
  output:<<"">>

=======================================================
  Failed: 1.  Skipped: 0.  Passed: 18.
error
```

(it says `expected` is `OOPS`).

**Comment by NobbZ** [comment]
Fixed by #155

**Comment by NobbZ** [comment]
Thank you for reporting and fixing this!
